src/DssSpell.sol

```diff
@@ -37,7 +75,37 @@ contract DssSpellAction is DssAction {
     // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/d8c711404e9e17db24b611aa03d45507c1993148/governance/votes/Executive%20vote%20-%20November%2016%2C%202022.md -q -O - 2>/dev/null)"
 
     string public constant override description =
-        "2022-11-16 MakerDAO Executive Spell | Hash: 0xf3023b0a9b3c423edf28bf4cdbd61d287ab29265d462a5fae0378227c153ea7b";
+        "2022-11-30 MakerDAO Executive Spell | Hash: TODO";
+
+    uint256 constant internal RAY = 10 ** 27;
+    uint256 constant internal RAD = 10 ** 45;
+    uint256 constant internal MILLION = 10 ** 6;
+
+    bytes32 constant internal ILK = "DIRECT-COMPV2-DAI";
+
+    address constant internal D3M_HUB = 0x12F36cdEA3A28C35aC8C6Cc71D9265c17C74A27F;
+    address constant internal D3M_MOM = 0x1AB3145E281c01a1597c8c62F9f060E8e3E02fAB;
+    address constant internal D3M_MOM_LEGACY = 0x99A219f3dD2DeEC02c6324df5009aaa60bA36d38;
+    address constant internal D3M_COMPOUND_POOL = 0x621fE4Fde2617ea8FFadE08D0FF5A862aD287EC2;
+    address constant internal D3M_COMPOUND_PLAN = 0xD0eA20f9f9e64A3582d569c8745DaCD746274AEe;
+    address constant internal D3M_ORACLE = 0x0e2bf18273c953B54FE0a9dEC5429E67851D9468;
```

Comment:
just to be acknowledged: d3m contracts looks deployed with optimizations on (200 runs) and tested with that (also in certora), even if most of the d3m contracts do not require optimizations on (except for the Plan) and some d3m components will be used for other d3m onboardings (like the Hub). 
Considering recent optimizer bugs been discovered in latest solc versions would be cautious to not use optimizations if not strictly required, even if the contracts in scope are not affected by those.

Reply:
I think we can resolve this one, contracts won't be redeployed as they followed the details used on its own repo.